threescale_api/auth.py

```diff
@@ -4,33 +4,35 @@
 
 import requests
 
+
```

Comment:
this renaming doesn't seem to be relevant to the change in this PR, rather shouldn't be in this PR

Reply:
Fixed